webapp/packages/core-localization/src/locales/en.ts

```diff
@@ -92,6 +92,7 @@ export default [
   ['ui_close_all_to_the_left', 'Close all to the Left'],
   ['ui_or', 'Or'],
   ['ui_download', 'Download'],
+  ['ui_upload', 'Upload'],
```

Comment:
We have ui_upload_file and ui_upload_files, do we need another one?

Reply:
they are different, but yep, it's a tricky case